### PR TITLE
Removed CakePHP

### PR DESCRIPTION
**Pull request by mtancoigne**
After all the discussions with @alayek, I put the CakePHP part in a [different repo](https://github.com/mtancoigne/zeus-php-cakephp3). 

So now the box creates an empty web development for PHP. [Here is the test following the README](https://hidden-taiga-91781.herokuapp.com/)

**Before merging**:

@byteknacker I'll need your help to integrate the `provision.sh` with Zeus : now the user should have two options : an empty PHP dev environment, or a CakePHP3 dev environment. If you can guide me on this, that would be perfect.

On my side, I have to polish the zeus-php-cakephp3 repo and test it.

### CODE REVIEW

File: php/vagrant/provision.sh [hunk 1]
```diff
@@ -63,7 +66,7 @@ apt-get install -y --no-install-recommends heroku-toolbelt ruby dos2unix man cur
 echo "ServerName ${SERVER}" >> /etc/apache2/apache2.conf;
 
 # Installing the PHP dev stack
-apt-get install -y --no-install-recommends git postgresql postgresql-contrib phpunit php5 php5-intl php5-pgsql php5-mcrypt php5-sqlite php5-apcu php5-cli
+apt-get install -y --no-install-recommends git postgresql postgresql-contrib phpunit php5 php5-intl php5-pgsql php5-mcrypt php5-sqlite php5-apcu php5-cli php5-gd
```
Comment (alayek) [inline php/vagrant/provision.sh:69]: You don't need `git`. `minimal/trusty64` comes pre-installed with `git`, `vim`, `opensshd`.

**Comment by alayek** [comment]
I did a vagrant up and tested that `php`, `postgres` etc. are in correct version.

Looks alright, but next time, also share how to test your PR.

# LGTM :+1:

Let's :shipit: :package:

**Comment by koustuvsinha** [comment]
LGTM :+1:

**Comment by mtancoigne** [comment]
Sorry about the tests part, had no time to write somethin else than the index.php file. It contains a big phpinfo button wich will return all the php dependencies and environment. I'll specify it in next PRs

**Pull request merged**